pkg/cache/clusterqueue.go

```diff
@@ -237,42 +247,53 @@ func (c *clusterQueue) inactiveReason() (string, string) {
 		return "Terminating", "Can't admit new workloads; clusterQueue is terminating"
 	case pending:
 		reasons := make([]string, 0, 3)
+		messages := make([]string, 0, 3)
 		if c.isStopped {
 			reasons = append(reasons, "Stopped")
+			messages = append(messages, "Is stopped.")
 		}
-		if c.hasMissingFlavors {
+		if len(c.missingFlavors) > 0 {
 			reasons = append(reasons, "FlavorNotFound")
+			messages = append(messages, fmt.Sprintf("References missing ResourceFlavor %v.", c.missingFlavors))
 		}
-		if c.hasMissingOrInactiveAdmissionChecks {
-			reasons = append(reasons, "CheckNotFoundOrInactive")
+		if len(c.missingAdmissionChecks) > 0 {
+			reasons = append(reasons, "AdmissionCheckNotFound")
```

Comment:
It would be nice to declare the Reasons for the ClusterQueueActive condition as consts. `apis/kueue/v1beta1/clusterqueue_types.go` seems the proper place, next to the const for the condition -analogously as we declare conditions for workloads. 

Since we didn't declare them before it could be a follow up.

Reply:
done